package Protos;

message LeakyIntegrateAndFireC {
    required double tau_m = 1;
    required double rest_pot = 2;
    required double tau_ref = 3;
    required double noise = 4;
}

message LeakyIntegrateAndFireState {
    required double p = 1;
    required double u = 2;
    required double ref_time = 3;
}

